docs-ref-services/site-recovery.md

```diff
@@ -21,15 +21,20 @@ Site Recovery allows you to automate replication of Azure VMs between regions, o
 
 Learn more about [Azure Site Recovery](https://docs.microsoft.com/en-us/azure/site-recovery/site-recovery-overview).
 
-## Management Package
+## Management package
 
 ### Install npm module
+
+Install the Azure recovery servics npm module

```

Comment:
Typo in services. Also, according to the template it should be npm package: https://microsoft-my.sharepoint.com/personal/buhollan_microsoft_com/_layouts/15/WopiFrame.aspx?sourcedoc=%7Bcd8cb267-e673-42b1-9d7b-0882e19caf27%7D